commands/alias.go

```diff
@@ -105,6 +109,8 @@ func alias(command *Command, args *Args) {
 			eval = `function git --wraps hub --description 'Alias for hub, which wraps git to provide extra functionality with GitHub.'
 	hub $argv
 end`
+		case "rc":
+			eval = "eval `{hub alias -s}"
```

Comment:
Does this syntax correctly support if the output of `hub alias -s` spans over multiple lines? In posix shells, we needed to use double quotes for that.